src/promptflow/tests/sdk_cli_test/unittests/test_run.py

```diff
@@ -94,6 +94,13 @@ def test_dot_env_resolve(self):
         run = load_run(source=source, params_override=[{"name": run_id}])
         assert run.environment_variables == {"FOO": "BAR"}
 
+    def test_run_invalid_flow_path(self):
```

Comment:
Also add a test to guard invalid remote flow error?

Reply:
Sure, let me add one